packages/block-library/src/social-links/social-link/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import metadata from './block.json';
+import save from './save';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Social link' ),
+	parent: [ 'core/social-links' ],
+	icon: 'share',
```

Comment:
`parent` and `icon` should be moved to `block.json` file.

Reply:
:+1: